redis/asyncio/cluster.py

```diff
@@ -614,9 +640,9 @@ async def execute_command(self, *args: EncodableT, **kwargs: Any) -> Any:
         if passed_targets and not self._is_node_flag(passed_targets):
             target_nodes = self._parse_target_nodes(passed_targets)
             target_nodes_specified = True
-            retry_attempts = 1
+            retry_attempts = 0
 
-        for _ in range(retry_attempts):
+        while True:
```

Comment:
Can we hit a case where this fails to exit and just sits here? Counter impl?

Reply:
changed